tests/unit/lambdaService.unitTest.ts

```diff
@@ -1,87 +1,90 @@
-            it("should throw an error", async () => {
-                const service = new LambdaService(new Lambda());
-                expect.assertions(2);
-                try {
-                    await service.validateInvocationResponse({Payload: "{}", StatusCode: 200});
-                } catch (e) {
-                    expect(e.message).toEqual("Lambda invocation returned bad data: {}.");
-                    expect(e).toBeInstanceOf(Error);
-                }
-            });
-        });
+    context("gets OK response and no body object in ", () => {
+      it("should throw an error", async () => {
+        const service = new LambdaService(new Lambda());
+        expect.assertions(2);
+        try {
+          await service.validateInvocationResponse({ Payload: "{}", StatusCode: 200 });
+        } catch (e) {
+          expect(e.message).toEqual("Lambda invocation returned bad data: {}.");
+          expect(e).toBeInstanceOf(Error);
+        }
+      });
+    });
 
-        context("gets good response", ()=> {
-            it("should return the payload body", async () => {
-                const service = new LambdaService(new Lambda());
-                expect.assertions(1);
-                try {
-                    const result = await service.validateInvocationResponse({Payload: "{\"statusCode\": 200, \"body\": \"It worked\"}", StatusCode: 200});
-                    expect(result).toEqual({ statusCode: 200, body: "It worked" });
-                } catch {
-                    // Should never reach
-                }
-            });
-        });
+    context("gets good response", () => {
+      it("should return the payload body", async () => {
+        const service = new LambdaService(new Lambda());
+        expect.assertions(1);
+        try {
+          const result = await service.validateInvocationResponse({ Payload: "{\"statusCode\": 200, \"body\": \"It worked\"}", StatusCode: 200 });
+          expect(result).toEqual({ statusCode: 200, body: "It worked" });
+        } catch {
+          // Should never reach
+        }
+      });
     });
+  });
 
 
-    context("Invoke", () => {
-        context("gets an error from the Lambda SDK", () => {
-            it("bubbles that error up", async () => {
-                AWSMock.mock("Lambda", "invoke", (params: any, callback: any) => {callback(new Error("Oh no"));});
-                const lambda = new AWS.Lambda();
-                const service = new LambdaService(lambda);
-                expect.assertions(1);
-                try {
-                    await service.invoke({FunctionName: "bob"});
-                } catch (e) {
-                    expect(e.message).toEqual("Oh no");
-                }
-                AWSMock.restore("Lambda");
-            });
+  context("Invoke", () => {
+    context("gets an error from the Lambda SDK", () => {
+      it("bubbles that error up", async () => {
+        AWSMock.mock("Lambda", "invoke", (params: any, callback: any) => {
```

Comment:
Oh, that's where I used that... *note to self: remove aws-sdk-mock*

Reply:
Nothing wrong with aws-sdk-mock, it's required for mocking secrets manager in another file